aemedge/styles/styles.css

```diff
@@ -235,30 +235,55 @@ main .button-container {
     border-radius: 0.5rem;
     font-style: normal;
     font-size: var(--udexTypographyNavigationMediumXLFontSize);
+    line-height: var(--udexTypographyNavigationLineHeight);
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
     min-height: 36px;
     padding: 9px 10px;
 
-    &:is(a):not(.primary, .secondary)::after {
-      content: '\203A';
-      position: absolute;
-      padding-inline: var(--udexSpacer4);
-      transform: scale(1.3);
-    }
-
-    &:hover {
-      color: var(--udexColorBlue9);
-    }
-
     &:any-link,
     button {
       border-radius: 0.5rem;
       border: 0.0625rem solid transparent;
       font-weight: normal;
     }
 
+    &:is(a):not(.primary, .secondary) {
+      display: inline-flex;
+      align-items: center;
+      padding: 0;
+      min-height: 0;
+
+      &:visited {
+        color: var(--udexColorBlue10);
+        text-decoration: underline;
+      }
+
+      &:hover {
+        color: var(--udexColorBlue9);
+        text-decoration: underline;
+      }
+
+      &:active {
+        color: var(--udexColorBlue11);
+        text-decoration: underline;
+      }
+
+      &::after {
+        box-sizing: border-box;
+        content: "";
+        display: inline-block;
+        height: 1em;
```

Comment:
1rem not 1em
<img width="285" alt="Screenshot 2024-04-16 at 16 33 24" src="https://github.com/urfuwo/hlx-test/assets/41549648/df96223a-97f6-4fbf-af0b-40e4b41b7d5b">